### PR TITLE
docs: minor changes to the aws using dynamo on ec2

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: docs/deploy/aws.md [hunk 1]
```diff
@@ -37,50 +40,41 @@ If you already have a database, take note of the connection string and skip to t
...
 AWS credentials, for DynamoDB, can also be provided via environment variables, as described in the [configuration reference](../reference/configuration.md#using-environment-variables)
 Please refer to [AWS documentation](https://aws.amazon.com/dynamodb/getting-started/) for further information on DynamoDB
 
 ## Installation Options
 
 ### On EC2
 
-#### With PostgreSQL
-1. Save the following configuration file as `config.yaml`:
+1. Edit and same the following configuration file as `config.yaml`:
```
Comment (itai-david) [inline docs/deploy/aws.md:52]:
```suggestion
1. Edit and save the following configuration file as `config.yaml`:
```

File: docs/deploy/aws.md [hunk 2]
```diff
@@ -90,10 +84,13 @@ Please refer to [AWS documentation](https://aws.amazon.com/dynamodb/getting-star
...
 
 ### On ECS
+
 To support container-based environments like AWS ECS, lakeFS can be configured using environment variables. Here are a couple of `docker run` 
 commands to demonstrate starting lakeFS using Docker:
+
 #### With PostgreSQL
-```sh
+
+```bash
```
Comment (itai-david) [inline docs/deploy/aws.md:93]: 1. What's the difference? (from markdown POV)
2. Why was it changed here and left as `sh` in the next section?

**Review by itai-david — CHANGES_REQUESTED** [review]
Hi.
Some minor question and a small typo below

File: docs/deploy/aws.md (hunk 2 above)
Comment (nopcoder) [reply docs/deploy/aws.md:93]: 1. bash is equal to sh syntax as far as I know - can revert
2. just added blank lines

**Review by itai-david — APPROVED** [review]

**Pull request merged**